### DISCUSSION TITLE
“快速复制”功能是否可以通过快捷键临时禁用?

### DISCUSSION THREAD

**Discussion by Caodaping** [shuax/MouseInc.Settings#89] [General]
感谢大佬开发如此强大的软件，这个软件对我的帮助非常大。

想请问一下，“快速复制”功能(鼠标选中或双击文字（根据光标）后自动复制)是否有办法临时禁用，比如按了什么快捷键。

使用场景：我经常需要校核Word文档，它是英中文对照的两列。有时候我不理解的时候，会选中一段英文，然后复制到浏览器中的翻译网站看一看。如果是翻译网站的翻译更贴切，我会把结果复制回来，粘贴回Word文件中修改。

多数时候，快速复制很有用，省却了按Ctrl+C。但偶尔想临时禁用，选中的目的是要移动内容(修改过程中)，而不希望复制，把剪贴板内容覆盖掉。

**Comment by shuax** [comment]
按住ctrl可以临时不复制

**Reply by Caodaping** [reply]
谢谢您，测试有效。

**Comment by Caodaping** [comment]
补充一下需求：
是否可以增加一个按键，按下时临时禁用复制功能，再按时恢复。

现在需要按住Ctrl键才能禁用该功能。有时候忙着的时候会忘记，有时候手指头不够用，因为还有一个辅助键盘需要按键.......谢谢!!

**Comment by shuax** [comment]
暂停恢复MouseInc | Ctrl+Alt+M
